server/routes/tickets.py

```
@@ -0,0 +1,670 @@
+
+            # Check permissions - clients can only see their tickets
+            if user_role == Role.CLIENT and ticket.client_id != user.id:
+                return restful_response(
+                    status="error", 
+                    message="Access denied", 
+                    status_code=403
+                )
+
+            # Get messages
+            messages = TicketMessage.query.filter_by(ticket_id=ticket_id)\
+                                         .order_by(TicketMessage.created_at.asc()).all()
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)